app/Http/Controllers/Admin/BanController.php

```diff
@@ -91,6 +91,12 @@ public function create(CreateBanRequest $request)
             ]);
         }
 
+        if (!$request->has('reason')) {
+            throw ValidationException::withMessages([
+                'reason' => 'You must provide a ban reason.',
+            ]);
+        }
```

Comment:
Please add this into `CreateBanRequest` instead, just using the `required` rule should do the trick